src/index.js

```diff
@@ -1 +1,10 @@
-alert("weather api");
\ No newline at end of file
+import fetchData from "./fetchData";
+// import loadDOM from "./loadDOM";
+
+
+document.addEventListener('click', (e) => {
```

Comment:
We shouldn't add the event listener to whole document here. Add it to the button element or to the form (with 'submit' event).

Also use `e.preventDefault()` to prevent the form to refresh the page.